src/interface_adapter/load_game/LoadGamePresenter.java

```diff
@@ -27,8 +29,32 @@ public void prepareView(LoadGameOutputData outputData) {
         roundState.setGenre(outputData.getGenre());
         roundState.setUserAnswer("");
 
+
+        if (outputData.getDifficulty().equalsIgnoreCase("easy")) {
+            roundState.setEasyRound(true);
+            roundState.setMediumRound(false);
+            roundState.setHardRound(false);
+            roundState.setMultipleChoice1(outputData.getMultipleChoiceAnswers().get(0));
+            roundState.setMultipleChoice2(outputData.getMultipleChoiceAnswers().get(1));
+        }
+        else if (outputData.getDifficulty().equalsIgnoreCase("Medium")) {
+            roundState.setEasyRound(false);
+            roundState.setMediumRound(true);
+            roundState.setHardRound(false);
+            roundState.setMultipleChoice1(outputData.getMultipleChoiceAnswers().get(0));
+            roundState.setMultipleChoice2(outputData.getMultipleChoiceAnswers().get(1));
+            roundState.setMultipleChoice3(outputData.getMultipleChoiceAnswers().get(2));
+            roundState.setMultipleChoice4(outputData.getMultipleChoiceAnswers().get(3));
```

Comment:
```suggestion
            roundState.setMultipleChoiceOptions(outputData.getMultipleChoiceAnswers());
```
Continuation of cleaning the round state